changelog/exitcodes.dd

```diff
@@ -0,0 +1,11 @@
+DUB command exit codes have been adjusted
```

Comment:
Instead of "adjust", which just hints at a change, perhaps something more positive ("are now more detailed")?